armi/reactor/reactors.py

```diff
@@ -292,8 +293,8 @@ def powerMultiplier(self):
     @property
     def lib(self):
         """"Get the microscopic cross section library."""
-        if self._lib is None:
-            runLog.info("Loading microscopic cross section library ISOTXS")
+        if self._lib is None and self._preloadCoreXS:
+            runLog.info(f"Loading existing ISOTXS microscopic cross section library")
```

Comment:
Should be a message here if lib is None and not preload, there should be a meaningful error pointing users to the preload setting when this starts failing on their previously-fine cases.

Reply:
Added the check and added a `ValueError`.